mrmustard/physics/representations.py

```diff
@@ -636,34 +627,32 @@ def __getitem__(self, idx: int | tuple[int, ...]) -> Fock:
 
     def __matmul__(self, other: Union[Bargmann, Fock]) -> Fock:
         r"""
-        Implements the inner product of ansatze across the marked indices.
-
-        If ``other`` is ``Fock``, the two representations are automatically reduced
-        before being contracted. The array of the returned representation has the largest
-        possible dimension along every axis.
-
-        If ``other`` is ``Bargmann``, it is converted to ``Fock`` before the contraction.
+        Implements the inner product of fock arrays over the marked indices.
+        ..code-block::
```

Comment:
doesn't really matter because it doesn't render, but just in case it gets moved one day
```suggestion
        Implements the inner product of fock arrays over the marked indices.

        .. code-block::
```